tests/braynsTestData.cpp

```diff
@@ -126,15 +126,9 @@ BOOST_AUTO_TEST_CASE(render_protein_in_stereo_and_compare)
 
     const char* app = testSuite.argv[0];
     const std::string pdbFile(BRAYNS_TESTDATA + std::string("1bna.pdb"));
-    const char* argv[] = {app,
-                          "--synchronous-mode",
-                          "on",
-                          "--accumulation",
-                          "off",
-                          "--pdb-file",
-                          pdbFile.c_str(),
-                          "--camera",
-                          "stereo"};
+    const char* argv[] = {
+        app,          "--synchronous-mode", "on",       "--accumulation", "off",
+        "--pdb-file", pdbFile.c_str(),      "--camera", "stereo"};
```

Comment:
cleanup whitespaces

Reply:
damn clangformat. It's shouldn't be my job :(

Reply:
Can't do it. clangformat likes it this way